packages/catalogue/examples/example.js

```diff
@@ -0,0 +1,58 @@
+import { credentialsProvider, init as initAuth } from '@tidal-music/auth';
+
+import { catalogueClient, initCatalogueClient } from '../dist';
+
+async function runExample(clientId, clientSecret) {
+  await initAuth({
+    clientId,
+    clientSecret,
+    credentialsStorageKey: 'clientCredentials',
+  });
+
+  await initCatalogueClient(credentialsProvider);
+
+  const { data, error } = await catalogueClient.GET('/albums/{id}', {
```

Comment:
Cool, so there are types for the `path`, right? So the user knows what can be requested and whats needed to pass in, right?

Reply:
That is correct!
Easiest to see in the IDE with this checked out :-)

Reply:
Works great had a quick go in the IDE!